javascript/chronometer.js

```diff
@@ -1,26 +1,47 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0; 
+    this.intervalId = 0; 
   }
   startClick(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+    this.currentTime += 1
+    // this is where i want to invoke the code //this callback will update the DOM
```

Comment:
just as the comment, yes, this is where you needed to invoke the callback function. simply adding: callback()